class/appcat-service-postgresql.yml

```
@@ -1,6 +1,24 @@
 parameters:
   kapitan:
+    dependencies:
+      - type: helm
+        source: ${appcat_service_postgresql:charts:appcat-service-postgresql:source}
+        chart_name: provider-postgresql
+        version: ${appcat_service_postgresql:charts:appcat-service-postgresql:version}
```

Please make the chart name configurable, so that it can be properly picked up by Renovate in the defaults.yaml, but also make it possible to define forks that have a different chart name.

---

Do you have an example for forks?

---

No, but I'm simply asking for something like this:
```suggestion
        source: ${appcat_service_postgresql:charts:appcat-service-postgresql:source}
        chart_name: ${appcat_service_postgresql:charts:appcat-service-postgresql:chart}
        version: ${appcat_service_postgresql:charts:appcat-service-postgresql:version}
```

---

Oh that was clear from the beginning, thank you.